messaging/types/store_node_batch.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "time"
+
+type StoreNodeBatch struct {
```

Comment:
Same here. Perhaps it's fine for now to use adapters to Waku as the first step, but I guess in the future we don't want such low-level types to be exposed by the Messaging API?

Reply:
Step by step https://github.com/status-im/status-go/pull/6585#issuecomment-2887558437